base-notebook/test/test_container_options.py

```diff
@@ -74,31 +74,6 @@ def test_sudo(container):
     assert 'uid=0(root)' in c.logs(stdout=True).decode('utf-8')
 
 
-def test_sudo_path(container):
-    """Container should include /opt/conda/bin in the sudo secure_path."""
-    c = container.run(
-        tty=True,
-        user='root',
-        environment=['GRANT_SUDO=yes'],
-        command=['start.sh', 'sudo', 'which', 'jupyter']
-    )
-    rv = c.wait(timeout=10)
-    assert rv == 0 or rv["StatusCode"] == 0
-    assert c.logs(stdout=True).decode('utf-8').rstrip().endswith('/opt/conda/bin/jupyter')
-
-
-def test_sudo_path_without_grant(container):
-    """Container should include /opt/conda/bin in the sudo secure_path."""
-    c = container.run(
-        tty=True,
-        user='root',
-        command=['start.sh', 'which', 'jupyter']
-    )
-    rv = c.wait(timeout=10)
-    assert rv == 0 or rv["StatusCode"] == 0
-    assert c.logs(stdout=True).decode('utf-8').rstrip().endswith('/opt/conda/bin/jupyter')
-
-
```

Comment:
I vaguely recall we used `su` or `gosu` in the past in these images and ran into problems reported in issues and corrected in past PRs. If we make this switch, I think we need to at least retain the spirit of the tests that were in place to check behavior with and without `GRANT_SUDO=yes`.

Reply:
> I vaguely recall we used su or gosu in the past in these images and ran into problems reported in issues and corrected in past PRs.
Could you give me a reference for that issues?